### PR TITLE
Add fields to sorting flaw api

### PR DESCRIPTION
**Pull request by costaconrado**
This PR includes additional fields into flaw API for sorting.

Closes OSIDB-2367.

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -3095,6 +3102,7 @@ paths:
             - affects__type
             - affects__updated_dt
             - affects__uuid
+            - bz_id
```
Comment (osoukup) [inline openapi.yml:3105]: I am either hallucinating or I see some items twice in the list :eyes: can you please double check it?

**Review by osoukup — APPROVED** [review]
LGTM but it seems to me that there are some duplicates in the list

File: openapi.yml (hunk 1 above)
Comment (JakubFrejlach) [reply openapi.yml:3105]: There is `-bz-id` and `bz-id` for descending and ascending order.

Comment (costaconrado) [reply openapi.yml:3105]: it adds all items twice one for asc and one for desc like: `bz_id` and `-bz_id` in this case.

Comment (osoukup) [reply openapi.yml:3105]: I see! The difference is not too visible for a tired human :smile:

**Pull request merged**